nixos/modules/services/network-filesystems/moosefs.nix

```diff
@@ -205,12 +233,13 @@ in {
       };
     };
 
-    # Create system user account for daemons
-    users = lib.mkIf ( cfg.runAsUser && ( cfg.master.enable || cfg.metalogger.enable || cfg.chunkserver.enable ) ) {
+    users = lib.mkIf ( cfg.runAsUser && ( cfg.master.enable || cfg.metalogger.enable || cfg.chunkserver.enable || cfg.cgiserver.enable ) ) {
       users.moosefs = {
         isSystemUser = true;
-        description = "moosefs daemon user";
+        description = "MooseFS daemon user";
         group = "moosefs";
+        home = "/var/lib/mfs";
+        createHome = true;
```

Comment:
Hardcoding `/var/lib/mfs` is actually not meaningful. The moment you set DATA_PATH in the config of any service this directory is unused. It is also not clear to me why we should set this as the home directory for the mfs user.